setup/install/providers/kubernetes-v2/aws-eks.md

```diff
@@ -177,24 +150,19 @@ data:
 
 ```
 
-Run following to join nodes with the cluster
+Run following to join nodes with the cluster:
 
-```
+```bash
 kubectl apply -f aws-auth-cm.yaml
 ```
 
-Watch the nodes to come in ready state
+Watch the status of your nodes and wait for them to reach the `Ready` status.
 
-```
+```bash
```

Comment:
Run this *after* the nodes reach ready state?

Reply:
added : in the end. This command itself identifies, if the nodes have reached the steady state